meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex-fsl-fw-utils_%.bbappend

```diff
@@ -0,0 +1 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Almost done with my rebase, but stumbled upon a problem with above change. When I use the `require u-boot-mender-toradex-nxp.inc` I get following error:

```
Log data follows:
| DEBUG: Executing python function sstate_task_prefunc
| DEBUG: Python function sstate_task_prefunc finished
| DEBUG: Executing shell function do_deploy
| /home/mirzak/project/mender.io/ubifs-support/build/tmp/work/colibri_vf-poky-linux-gnueabi/u-boot-toradex-fsl-fw-utils/v2016.11-v2.6.2b1+gitAUTOINC+b5f6ee6ab1-r0/temp/run.do_deploy.3352: 104: /home/mirzak/project/mender.io/ubifs-support/build/tmp/work/colibri_vf-poky-linux-gnueabi/u-boot-toradex-fsl-fw-utils/v2016.11-v2.6.2b1+gitAUTOINC+b5f6ee6ab1-r0/temp/run.do_deploy.3352: do_deploy: not found
| WARNING: exit code 127 from a shell command.
| ERROR: Function failed: do_deploy (log file is located at /home/mirzak/project/mender.io/ubifs-support/build/tmp/work/colibri_vf-poky-linux-gnueabi/u-boot-toradex-fsl-fw-utils/v2016.11-v2.6.2b1+gitAUTOINC+b5f6ee6ab1-r0/temp/log.do_deploy.3352)
ERROR: Task (/home/mirzak/project/mender.io/ubifs-support/layers/meta-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex-fsl-fw-utils_git.bb:do_deploy) failed with exit code '1'
```

A bit strange, it creates the do_deploy task but does not populate anything in to it. Anything you seen before?